diff -ur postgresql-14.2_original/src/interfaces/libpq/Makefile postgresql-14.2/src/interfaces/libpq/Makefile
--- postgresql-14.2_original/src/interfaces/libpq/Makefile	2022-02-23 09:31:25.000000000 +0100
+++ postgresql-14.2/src/interfaces/libpq/Makefile	2022-02-23 10:18:14.000000000 +0100
@@ -96,8 +96,6 @@
 
 PKG_CONFIG_REQUIRES_PRIVATE = libssl libcrypto
 
-all: all-lib
-
 # Shared library stuff
 include $(top_srcdir)/src/Makefile.shlib
 backend_src = $(top_srcdir)/src/backend
